androidClient/app/src/main/res/layout/item_format_herald.xml

```diff
@@ -4,91 +4,76 @@
+    <ImageButton
+        android:id="@+id/item_format_herald_share"
+        android:layout_width="44dp"
+        android:layout_height="44dp"
+        android:background="@color/transparent"
         app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:placeholderImage="@drawable/article_icon"
-        app:roundAsCircle="true" />
+        app:layout_constraintTop_toBottomOf="@+id/item_format_herald_title"
+        app:srcCompat="@drawable/ic_share" />
 
     <com.like.LikeButton
         android:id="@+id/item_format_herald_heart"
         android:layout_width="44dp"
         android:layout_height="44dp"
-        android:layout_marginTop="8dp"
         app:anim_scale_factor="2.5"
         app:circle_end_color="@color/likeCircleEnd"
         app:circle_start_color="@color/likeCircleStart"
         app:dots_primary_color="@color/likeDotsColorPrimary"
         app:dots_secondary_color="@color/likeDotsColorSecondary"
         app:icon_size="22dp"
         app:icon_type="heart"
-        app:layout_constraintLeft_toLeftOf="@+id/item_format_herald_image"
-        app:layout_constraintTop_toBottomOf="@+id/item_format_herald_image"
+        app:layout_constraintRight_toLeftOf="@id/item_format_herald_share"
+        app:layout_constraintTop_toBottomOf="@+id/item_format_herald_title"
         app:liked="false" />
 
-    <ImageButton
-        android:id="@+id/item_format_herald_share"
-        android:layout_width="44dp"
-        android:layout_height="44dp"
-        android:background="@color/transparent"
-        app:layout_constraintBottom_toBottomOf="@+id/item_format_herald_heart"
-        app:layout_constraintRight_toRightOf="@+id/item_format_herald_image"
-        app:layout_constraintTop_toTopOf="@+id/item_format_herald_heart"
-        app:srcCompat="@drawable/ic_share" />
-
     <TextView
         android:id="@+id/item_format_herald_date"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_marginEnd="8dp"
-        android:layout_marginLeft="16dp"
-        android:layout_marginRight="8dp"
-        android:layout_marginStart="16dp"
-        android:layout_marginTop="8dp"
+        android:layout_marginTop="12dp"
+        android:fontFamily="@font/roboto_regular"
         android:maxLines="1"
+        android:paddingStart="16dp"
         android:textColor="?customHeraldDateTextColor"
-        android:textSize="12sp"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toLeftOf="@+id/item_format_herald_image"
+        android:textSize="14sp"
         app:layout_constraintTop_toBottomOf="@+id/item_format_herald_title" />
-
-    <TextView
-        android:id="@+id/item_format_herald_desc"
-        android:layout_width="0dp"
-        android:layout_height="0dp"
-        android:layout_marginEnd="8dp"
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="0.5dp"
```

Comment:
https://stackoverflow.com/a/21746197 suggests `1px` for these use cases.

Reply:
I'll make a final commit. Do u have any other suggestions. Please Acknowledge this message accordingly.

Reply:
seems ok